counting-sheep.py

```diff
@@ -0,0 +1,7 @@
+def count_sheeps(arrayOfSheeps):
+    """With True meaning sheep present, count the number of sheep in a list."""
+    count = 0
+    for i in arrayOfSheeps:
+    if i == True:
+        count += 1
+    return count
```

Comment:
This will work; as you get better with Python, think about how you could write something like this in one line using the `sum` builtin function.